versions-prod.cfg

```diff
@@ -28,7 +28,7 @@ WebTest = 3.0.0
 ZConfig = 3.6.0
 ZEO = 5.3.0
 ZODB = 5.7.0
-cryptography = 37.0.2
+cryptography = 37.0.3
```

Comment:
Please revert, `cryptography` was yanked due to a security issue, see https://github.com/zopefoundation/Zope/issues/1049

Reply:
Oops, I should have noticed that. Fixed now.